packages/meteor/asl-helpers.js

```diff
@@ -4,7 +4,8 @@ const updateAslStore = (key, value) => getAslStore()[key] = value;
 
 const bootstrap = global.__meteor_bootstrap__; 
 
-Meteor.isFibersDisabled = !!(bootstrap && bootstrap.isFibersDisabled);
+// From 3.0 onwards, fibers is disabled by default (tools may not have bootstrap defined).
+Meteor.isFibersDisabled = !bootstrap || bootstrap.isFibersDisabled !== false;
```

Comment:
When when running as part of a build plugin in Meteor 2, bootstrap.isFibersDisabled will be undefined (it was added in Meteor 3). It should view undefined as meaning fibers are enabled.

Reply:
@zodern Yeah, debugged a bit more here and found out that the problem is that when running on the tools the `global.__meteor_bootstrap__` is undefined, while the `__meteor_bootstrap__` is not. Now I fixed and it's the behavior you described now, if the bootstrap is undefined, fibers will be enabled.